src/v2/Components/Modal/ModalWrapper.tsx

```diff
@@ -137,7 +138,13 @@ export class ModalWrapper extends React.Component<
       <Theme>
         <Wrapper isShown={isShown || isAnimating}>
           <GlobalStyle suppressMultiMountWarning />
-          {isShown && <ModalOverlay onClick={this.close} />}
+          {isShown && (
+            <ModalOverlay
+              onClick={
+                !this.props.disableCloseOnBackgroundClick ? this.close : null
```

Comment:
#superminor - its a bit more readable through time and slightly less complex to lead with the `true` case versus taking the true case and inverting it and then evaluating: 

```tsx
this.props.disableCloseOnBackgroundClick ? null : this.close
```